editor/src/clj/editor/ui.clj

```diff
@@ -1213,19 +1213,14 @@
   {:control control
    :menu-id menu-id})
 
-(defn- wrap-menu-image [node]
-  (doto (Pane.)
-    (children! [node])
-    (add-style! "menu-image-wrapper")))
```

Comment:
Not sure what was the significance of this wrapper, there was no styling associated with `.menu-image-wrapper` style class. I removed it to simplify CSS styling.